<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="text_writes">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>256</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Writes</string>
    </property>
   </widget>
   <widget class="MemoryView" name="scroll_memory" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>771</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>771</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_mem_view">
       <property name="text">
        <string>Memory view</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbl_addr">
       <property name="text">
        <string>Selected addres:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lned_addr">
       <property name="text">
        <string>0000</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>185</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_run">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_step">
         <property name="text">
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_speed">
         <property name="text">
          <string>Speed (Hz)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="speed">
         <property name="maximum">
          <number>50000000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_regA">
           <property name="text">
            <string>Register A</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regA">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regX">
           <property name="text">
            <string>Register X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regX">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regL">
           <property name="text">
            <string>Register L</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regL">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="lbl_regB">
           <property name="text">
            <string>Register B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regB">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regS">
           <property name="text">
            <string>Register S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regS">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regT">
           <property name="text">
            <string>Register T</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regT">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lbl_regF">
           <property name="text">
            <string>Register F</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regF">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regPC">
           <property name="text">
            <string>Register PC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regPC">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_regSW">
           <property name="text">
            <string>Register SW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lned_regSW">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lned_float_dec">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_float_dec">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Float in decimal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="menu_load"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_load">
   <property name="text">
    <string>Load obj</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MemoryView</class>
   <extends>QWidget</extends>
   <header>memoryview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
